app/_how-tos/kic-plugin-authentication.md

```diff
@@ -0,0 +1,212 @@
+
+prereqs:
+  kubernetes:
+    gateway_api: true
+  entities:
+    services:
+      - echo-service
+
+cleanup:
+  inline:
+    - title: Uninstall KIC from your cluster
+      include_content: cleanup/products/kic
+      icon_url: /assets/icons/kubernetes.svg
+---
+
+## Allowing multiple authentication methods
+
+The default behavior for Kong authentication plugins is to require credentials for all requests even if a request has been authenticated through another plugin. Configure an anonymous consumer on your authentication plugins to set clients authentication options.
+
+## Create consumers
+
+Create two consumers that use different authentication methods:
+
+* `consumer-1` uses `basic-auth`
+* `consumer-2` uses `key-auth`
+
+1. Create a secret to add a `basic-auth` credential for `consumer-1`:
+
+    ```bash
+    echo '
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: consumer-1-basic-auth
+      namespace: kong
+      labels:
+        konghq.com/credential: basic-auth
+    stringData:
+        username: consumer-1
+        password: consumer-1-password
+    ' | kubectl apply -f -
+    ```
+
+1. Create a secret to add a `key-auth` credential for `consumer-2`:
+
+    ```bash
+    echo '
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: consumer-2-key-auth
+      namespace: kong
+      labels:
+        konghq.com/credential: key-auth
+    stringData:
+      key: consumer-2-password
+    ' | kubectl apply -f -
+    ```
+
+1.  Create a consumer named `consumer-1`:
+
+{% entity_example %}
+type: consumer
+data:
+  username: consumer-1
+  credentials:
+    - consumer-1-basic-auth
+  
+indent: 4
+{% endentity_example %}
+
+1.  Create a consumer named `consumer-2`:
+
+{% entity_example %}
+type: consumer
+data:
+  username: consumer-2
+  credentials:
+    - consumer-2-key-auth
+  
+indent: 4
+{% endentity_example %}
+
+## Secure the service
+
+Once the Consumers and credentials are created, you can add authentication plugins to your Service.
+
+First, create a `key-auth` plugin. Notice the `anonymous` configuration option, which means "if no credentials match, assign the consumer named `anonymous` to the request":
+
+{% entity_example %}
+type: plugin
+data:
+  name: key-auth
+  config:
+    anonymous: anonymous
+  
+  service: echo
+  skip_annotate: true
+{% endentity_example %}
+
+As Consumer 2 is using `basic-auth`, we also need to create a `basic-auth` plugin:
+
+{% entity_example %}
+type: plugin
+data:
+  name: basic-auth
+  config:
+    anonymous: anonymous
+  
+  service: echo
+  other_plugins: key-auth
+{% endentity_example %}
+
+## Create an anonymous consumer
+
+Your endpoints are now secure, but neither consumer can access the endpoint when providing valid credentials. This is because each plugin will verify the consumer using it’s own authentication method.
```

Comment:
```suggestion
Your endpoints are now secure, but neither Consumer can access the endpoint when providing valid credentials. This is because each plugin will verify the Consumer using it’s own authentication method.
```